pkg/alertmanager/statefulset.go

```diff
@@ -200,6 +205,24 @@ func makeStatefulSetSpec(a *monitoringv1.Alertmanager, config Config) (*v1beta1.
 		},
 	}
 
+	var livenessProbe *v1.Probe
+	var readinessProbe *v1.Probe
+	if !a.Spec.ListenLocal {
```

Comment:
why removing the probe?

Reply:
probes on loopback don't seem to work, so I decided it's the responsibility of the "entry point" container